pontoon/pretranslation/tests/test_pretranslate.py

```diff
@@ -154,21 +151,21 @@ def test_get_pretranslations_fluent_accesskeys_no_attribute_source(
         """
         title =
             .foo = gt_translation
-            .accesskey = gt_translation
+            .accesskey = B
```

Comment:
Accesskeys should never go through machine translation, and so this is left untouched as the message does not have a value or attribute from which we can look up what this is an accesskey of.

Reply:
Ah, sorry - pretranslation / machine translation confusion on my end.